x-pack/plugins/security_solution/public/common/components/links/index.tsx

```diff
@@ -334,9 +340,10 @@ const CaseDetailsLinkComponent: React.FC<CaseDetailsLinkComponentProps> = ({
       return navigateToApp(APP_UI_ID, {
         deepLinkId: SecurityPageName.case,
         path: getCaseDetailsUrl({ id: detailName, search }),
+        openInNewTab,
```

Comment:
can you add the test case if not for the CaseDetailsLink but at least to your change?

Reply:
I added a test case for our code already (see [here](https://github.com/elastic/kibana/pull/197494/files#diff-f36e2826970fba00221456ec30e4c63e79599f1d55f2f1030ca59bd6ab7fffddR102)).

I was trying to add a test for this component but realized it's not being tested at all currently. Do you want me to add a series of tests for it?

Reply:
again I am not familiar with conventions yet, but whenever I touch functionality especially in shared code I try to add  some coverage. I know it's not your responsibility to add all the tests for a component that hasn't been covered from the start, that's why I said only your case. But if there is no convention covering that. Feel free to ignore, up to you.

Reply:
haha we answered at the same time 😆